internal/util/validation/validation.go

```diff
@@ -70,6 +71,28 @@ func ValidateString(s *string, path string, minLen int, maxLen int, patternRegex
 	return asErrors(errs)
 }
 
+func ValidateBase64Field(field *string, maxLenMB int) []error {
+	var errs []error // Initialize an empty slice of errors
+
+	if field == nil {
+		errs = append(errs, fmt.Errorf("field is nil"))
+	} else {
+		if len(*field) == 0 {
+			errs = append(errs, fmt.Errorf("field is empty"))
+		}
+		maxLen := maxLenMB * 1000000
+		if len(*field) > maxLen {
+			errs = append(errs, fmt.Errorf("field exceeds maximum length of %d", maxLen))
```

Comment:
Using the `field.ErrorList` type as in the other functions (e.g. `errs = append(errs, field.TooLong(fieldPathFor(path), s, maxLen))`) would give more consistent validation output.

Reply:
I didn't use field.ErrorList on purpose because it's from the K8s apimachinery, which might not make sense in this context?

Reply:
The logic is already used by the other validation functions in this package. It provides the helpful information _where_ in the manifest the validation error is and promotes that the same kind of error is reported the same way. And you're not using it _on purpose_?